src/viur/core/modules/history.py

```diff
@@ -0,0 +1,742 @@
+import difflib
+import enum
+import json
+import logging
+import typing as t
+from viur.core import db, conf, utils, current, tasks
+from viur.core.render.json.default import CustomJsonEncoder
+from prototypes.skeleton import SkeletonAbstractSkel, SkeletonInstance
+from viur.core.prototypes.list import List
+from viur.core.bones import *
+from bones import *  # overwrites UserBone
+from google.cloud import bigquery, exceptions
+
+
+class ViurHistorySkel(SkeletonAbstractSkel):
+    """
+    Skeleton used for a ViUR history entry to log any relevant changes
+    in other Skeletons.
+
+    The ViurHistorySkel is also used as the base for a biquery logging table,
+    see below.
+    """
+
+    kindName = "viur-history"
+    creationdate = changedate = None
+
+    version = NumericBone(
+        descr="Version",
+    )
+
+    action = StringBone(
+        descr="Action",
+    )
+
+    tags = StringBone(
+        descr="Tags",
+        multiple=True,
+    )
+
+    timestamp = DateBone(
+        descr="Zeitstempel",
+        defaultValue=lambda *args, **kwargs: utils.utcNow(),
+        localize=True,
+    )
+
+    user = UserBone(
+        updateLevel=RelationalUpdateLevel.OnValueAssignment,
+        searchable=True,
+        refKeys=[
+            "key",
+            "name",
+            "lastname",
+            "firstname",
+            "personnelnumber",
+            "last_audit",
+            "company",
```

Comment:
These bones does not exist in the `UserSkel`
```suggestion
```

Please remove them from the  related mapping and schema too